addons/website_event_track/data/mail_template_data.xml

```diff
@@ -29,4 +29,163 @@
+<t t-set="event_organizer" t-value="object.event_id.organizer_id"/>
+<t t-set="event_address" t-value="object.event_id.address_id"/>
+<t t-set="registration_ids" t-value="object.ids if not is_sale else object._get_event_registration_ids_from_order()"/>
+<table border="0" cellpadding="0" cellspacing="0"  width="590" style="padding: 16px; background-color: white; color: #454748; border-collapse:separate;">
+<tbody>
+    <!-- HEADER -->
+    <tr>
+        <td align="center" style="min-width: 590px;">
+            <table width="590" border="0" cellpadding="0" cellspacing="0" style="min-width: 590px; background-color: white; padding: 0px 8px 0px 8px; border-collapse:separate;">
+                <tr><td valign="middle">
+                    <span style="font-size: 10px;">Your Event Track</span><br/>
+                    <span style="font-size: 20px; font-weight: bold;" t-out="object.name"/>
+                </td><td valign="middle" align="right">
+                    <t t-if="not object.company_id.uses_default_logo">
+                        <img t-att-src="'/logo.png?company=%s' % object.company_id.id" style="padding: 0px; margin: 0px; margin-right: 10px; height: auto; width: 80px;" t-att-alt="'%s' % object.company_id.name"/>
+                    </t>
+                </td></tr>
+                <tr><td colspan="2" style="text-align:center;">
+                  <hr width="100%" style="background-color:rgb(204,204,204);border:medium none;clear:both;display:block;font-size:0px;min-height:1px;line-height:0; margin:16px 0px 16px 0px;"/>
+                </td></tr>
+            </table>
+        </td>
+    </tr>
+    <!-- TALK DESCRIPTION -->
+    <tr>
+        <td align="center" style="min-width: 590px;">
+            <table width="590" border="0" cellpadding="0" cellspacing="0" style="min-width: 590px; background-color: white; padding: 0px 8px 0px 8px; border-collapse:separate;">
+                <tr><td valign="top" style="font-size: 14px;">
+                    <div>
+                        Hello<t t-if="not user_id.is_public" t-out="' ' + user_id.name"/>,
+                        <br/><br/>
+                        You recently show interest for the talk
+                        <a t-if="object.description or object.partner_biography" t-attf-href="{{ object.get_base_url() }}{{ object.website_url }}" style="font-weight:bold;color:#875A7B;text-decoration:none;" t-out="object.name or ''">OpenWood Collection Online Reveal</a>
+                        <span t-else="" style="font-weight:bold;" t-out="object.name or ''">OpenWood Collection Online Reveal</span>.
+                    </div>
+                    <div>
+                        <br />
+                        <strong>Add this talk to your calendar</strong>
+                        <a t-att-href="google_url" style="padding:3px 5px;border:1px solid #875A7B;color:#875A7B;text-decoration:none;border-radius:3px;" target="new">
```

Comment:
Should have t-if, as people may use / render this template outside of the expected context, and it will crash.

Side quest: have those computed inside the template, like
```
<t t-set="agenda_urls" t-value="object._get_event_track_resource_urls()"/>
```
(this might require to set _get_event_track_resource_urls public, to check a bit if it is a good idea)